docs/BUILDING_WINDOWS.md

```diff
@@ -0,0 +1,73 @@
+# Building Habitat from source
+
+*NOTE:* This document is likely to change a lot over the coming months.
+
+*NOTE:* For Rust editing, I'd recommend Visual Studio Code and have [a blog post on how to set up the best Rust editing experience](http://stevenmurawski.com/powershell/2016/08/rustify).
```

Comment:
Thanks @reset 

Turns out that VS Code has pretty good rust support (not just on Windows), but I didn't want to make that assumption for folks in the doc.